YARCH/Sources/Modules/CatalogDetails/View/CatalogDetailsCoinImageView.swift

```diff
@@ -0,0 +1,23 @@
+// ImageView, которая может находиться в режиме плейсхолдера
+
+import UIKit
+
+class CatalogDetailsCoinImageView: UIImageView {
+
+    override var image: UIImage? {
+        didSet {
+            if image != nil {
+                removePlaceholder()
+            }
+        }
+    }
+
+    func becomePlaceholder() {
```

Comment:
Maybe it's better to use `showPlaceholder` and `hidePlaceholder` methods?

Reply:
Fixed